src/uct/api/uct.h

```diff
@@ -269,6 +269,8 @@ struct uct_iface_attr {
             size_t           max_zcopy;  /**< Maximal size for put_zcopy (total
                                               of @ref uct_iov_t::length of the
                                               @a iov parameter) */
+            size_t           opt_zcopy_align; /**< Alignment parameter */
```

Comment:
We need to document better. Is the **required** alignment or recommended for better performance or something else ?